### ISSUE TITLE
Density estimation trees

### ISSUE THREAD

**Issue by david-cortes** [pridiltal/ctv-AnomalyDetection#7]
MLPack for R is now on CRAN (https://cran.r-project.org/web/packages/mlpack/index.html) and contains an implementation of "density estimation trees" (paper: https://dl.acm.org/doi/abs/10.1145/2020408.2020507).

**Comment by pridiltal** [comment]
Thank you very much @david-cortes  for this suggestion. Does it have functions mainly focusing on anomaly detection problem? I checked the `det` function as you have suggested here. The main focus of this function is on density estimation with Density Estimation Trees. We can use that idea to find anomalous points (the values with the lowest densities). I am wondering whether this package contains any specific functions with anomaly detection capabilities.

Once again thank you very much for your support.

**Comment by david-cortes** [comment]
No, `mlpack` a general machine learning framework which contains a number of algorithms for e.g. regression, classification, clustering and others, among which it has the density estimation trees from that paper.
